crates/icrate/Cargo.toml

```diff
@@ -85,7 +85,10 @@ required-features = [
 ]
 
 [features]
-default = ["std", "apple"]
+default = ["std", "apple", "link"]
+
+# Link against framework, otherwise the correct framework should be load at runtime
+link = []
```

Comment:
I think I'd prefer this feature to be called `no-link`, since it is a thing that basically only binaries would be concerned with enabling. E.g. a library like `winit` would not want to touch this feature at all, but users of `winit` might still want the functionality.

We could consider having features `no-link-Foundation`, `no-link-AppKit` and so on instead, since that way it would be less of a breaking change if some library started depending on e.g. `AppKit`.
Alternatively, it should be clearly documented that enabling the `no-link` feature opts out of certain SemVer guarantees.

Reply:
Done.